src/plugins/colorchooser/structure_plugin.js

```diff
@@ -138,29 +141,29 @@ define( ['structure_custom', 'css!plugins/colorchooser/farbtastic/farbtastic.css
     switch( wData.address[ ga ][2] )
     {
       case 'r':
-        wData.bus_r = value;
+        wData.bus_r = value * 100 / 255.0;
```

Comment:
Only looking at the code (not really trying it): this seems wrong.

Here you are assuming value in the range of [0...255] and map it to [0...100](%) - and in line 146 below you are assuming value to be in the range of [0...100] and map it to [0...255]